doc/radosgw/layout.rst

```diff
@@ -103,7 +103,7 @@ is controlled by a 'policy' setting.[3]
 An RGW object may consist of several RADOS objects, the first of which
 is the head that contains the metadata, such as manifest, ACLs, content type,
 ETag, and user-defined metadata. The metadata is stored in xattrs.
-The head may also contain up to 512 kilobytes of object data, for efficiency
+The head may also contain up to 4 MB of object data, for efficiency
```

Comment:
how about just reference `rgw_max_chunk_size` like
```rst
The head may also contain up to :confval:`rgw_max_chunk_size` of object data, for efficiency
```

Reply:
and add an entry in the first section `doc/radosgw/config-ref.rst`, like 
```rst
.. confval:: rgw_max_chunk_size
```

Reply:
Wow, that was fast :D 

Excellent idea, I didn't know we could do that.  I'll be sure to use this method in future PRs.  I really like the work you've been doing to improve the structure of the options constellation.